### DISCUSSION TITLE
Export as Annotated CSV

### DISCUSSION THREAD

**Discussion by OGJakearroo** [alexjustesen/speedtest-tracker#1881] [Ideas]
I think this might be niche, but given you can already export as a CSV i would think this would be possible?

Is there any way we can get support for exporting to InfluxDBv2's annotated CSV format? https://docs.influxdata.com/influxdb/cloud/reference/syntax/annotated-csv/#annotated-csv-in-flux

Edit: Its been said its unsupported before, but not sure if you ever thought about direct exporting to their format :)

**Comment by alexjustesen** [comment]
https://github.com/alexjustesen/speedtest-tracker/pull/1866 will be in the next release, does this work if you're just after loading history into Influxdb?

**Reply by OGJakearroo** [reply]
It definately should.

As long as the cells are labelled the same, it should(tm) just work.

**Reply by alexjustesen** [reply]
There are changes which I am calling breaking in that PR, my initial implementation a couple of years ago was pretty basic and wasn't strongly typed. When that's released I strongly suggest re-stating your influxdb history.
